sigmastate/src/main/scala/sigmastate/lang/SigmaParser.scala

```diff
@@ -25,33 +23,34 @@ object SigmaParser extends Exprs with Types with Core {
   override def srcCtx(parserIndex: Int): SourceContext =
     SourceContext.fromParserIndex(parserIndex, currentInput.value)
 
-  val TmplBody = {
-    val Prelude = P( (Annot ~ OneNLMax).rep )
-    val TmplStat = P( Prelude ~ BlockDef | StatCtx.Expr )
-    P( "{" ~/ BlockLambda.? ~ Semis.? ~ TmplStat.repX(sep = Semis) ~ Semis.? ~ `}` )
+  def TmplBodyPrelude[_:P] = P( (Annot ~ OneNLMax).rep )
```

Comment:
ScalaDocs missed in the trait

Reply:
fixed in https://github.com/ScorexFoundation/sigmastate-interpreter/commit/43101550b16d94172e92a8fcea458a6cfdbe6557